packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "ins-c-component_invalid-component": {
```

Comment:
The names of classes should not be specific to HCC. CSS in JS also create unique hashes for class names co we don't have to worry about having conventions that much.

@fhlavac do we have some convention for this repo?

Reply:
let's omit the "ins-c-" prefix

classNames should start with the component name and then follow with more details to describe its purpose, so something like
"invalid-object" for the component wrapper, "invalid-object-text" for a text nested in that component, and so on

I will enhance our contribution guide with this info

Reply:
I was adding pfcg instead of ins-c @fhlavac I can drop that if that's what we decide.  Also should we now be using camel case?

Reply:
@dlabaj yes please, sorry for the changes. Other components should be already using it